slatedb-py/python/slatedb/slatedb.pyi

```diff
@@ -38,6 +38,46 @@ class SlateDB:
             ValueError: If the key is empty or there's a database error
         """
         ...
+
+    def put_int(self, key: bytes, value: int) -> None:
```

Comment:
I prefer we leave these new put_* and get_* out. I would rather the official SlateDB Python SDK directly reflect the underlying Rust SDK (which doesn't have these methods). My suggestion is to put helpers like this in your Python library/code.

Reply:
removed it, please review again